cmd/boot-script-service/main.go

```diff
@@ -417,9 +419,11 @@ func parseCmdLine() {
 	flag.StringVar(&bssdbName, "postgres-dbname", bssdbName, "(BSS_DBNAME) Postgres database name")
 	flag.StringVar(&sqlUser, "postgres-username", sqlUser, "(BSS_DBUSER) Postgres username")
 	flag.StringVar(&sqlPass, "postgres-password", sqlPass, "(BSS_DBPASS) Postgres password")
+	flag.StringVar(&jwksURL, "jwks-url", jwksURL, "Set the JWKS URL to fetch the public key for authorization")
```

Comment:
Similar to the comment regarding `retryAttempts`, we should also create an environment variable for this flag.